### PR TITLE
just init & modify some words

### PR DESCRIPTION
**Pull request by 1dooooo**
init  backend controller & change name for uri

### CODE REVIEW

File: REQUIREMENTS.MD [hunk 1]
```diff
@@ -91,7 +91,7 @@ CodeSegment {
 
 - `/index`，`GET`，首页文件
 
-- `/codeseg/all?language=xx&pageCount=xx&pageSize=xx`，`GET`，参数格式`language=xx`，无参数时获取所有代码段数据，有参数时，根据`language`参数值返回某种语言的代码段，根据`pageCount`和`pageSize`获取某一页的数据 如`/codeseg/all?language=Java*&pageCount=1&pageSize=10`返回Java语言的前十个代码段，返回值为JSON数组。响应体示例如下
```
Comment (EricJeffrey) [inline REQUIREMENTS.MD:94]: Interface change... lucky I have not started any related programming🙄

File: backend/src/main/java/com/thecheatsheet/cheatsheet/entity/CodeSegmentsEntity.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+@Getter
+@ToString
+public class CodeSegmentsEntity {
+
+    String id;
+    String title;
+    String desc;
+    String code;
+    String language;
+    String createAt;
```
Comment (EricJeffrey) [inline backend/src/main/java/com/thecheatsheet/cheatsheet/entity/CodeSegmentsEntity.java]: Should not `createAt` and `lastModify` be `long` type, they are timestamps, which is usually a 64bit integer.

File: backend/src/test/java/com/thecheatsheet/cheatsheet/CheatSheetApplicationTests.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package com.thecheatsheet.CheatSheet;
+package com.thecheatsheet.cheatsheet;
```
Comment (EricJeffrey) [inline backend/src/test/java/com/thecheatsheet/cheatsheet/CheatSheetApplicationTests.java:1]: Yeah I've noticed this either, package name should be lowercase, cool

**Review by EricJeffrey — COMMENTED** [review]
Just one thing about timestamp, what do you say?

**Comment by EricJeffrey** [comment]
Also I'm now investigating usage of Elasticsearch, for now the `paging` and `sorting` by language is totally feasible, and I'm thinking to make it a container using docker, but got some problems.

File: backend/src/main/java/com/thecheatsheet/cheatsheet/entity/CodeSegmentsEntity.java (hunk 2 above)
Comment (1dooooo) [reply backend/src/main/java/com/thecheatsheet/cheatsheet/entity/CodeSegmentsEntity.java]: indeed

**Comment by EricJeffrey** [comment]
Cool

**Pull request merged**